### PR TITLE
Cluster

### PR DESCRIPTION
**Pull request by marianogonzalez**

### CODE REVIEW

File: core/src/main/java/org/mule/runtime/core/internal/processor/interceptor/ReactiveInterceptorAdapter.java [hunk 1]
```diff
@@ -226,12 +225,10 @@ protected InternalEvent setInternalParamsForNotParamResolver(Component component
           resolvedParameters.putAll(params.entrySet().stream()
               .collect(toMap(e -> e.getKey(),
                              e -> new DefaultProcessorParameterValue(e.getKey(), null, () -> e.getValue().get()))));
-          Map<String, Object> interceptionEventParams = new HashMap<>();
-          interceptionEventParams.put(INTERCEPTION_RESOLVED_CONTEXT, context);
-          interceptionEventParams.put(INTERCEPTION_RESOLVED_PARAMS, resolvedParameters);
-          interceptionEventParams.put(INTERCEPTION_COMPONENT, component);
 
-          builder.internalParameters(interceptionEventParams);
+          builder.addInternalParameter(INTERCEPTION_RESOLVED_CONTEXT, context);
```
Comment (elrodro83) [inline core/src/main/java/org/mule/runtime/core/internal/processor/interceptor/ReactiveInterceptorAdapter.java]: this does not belong in this PR

**Comment by marianogonzalez** [comment]
--validate
